### PR TITLE
Extrapolation rule fixes

### PR DESCRIPTION
**Pull request by lennybronner**
## Description
Just added some logging and also a small fix for when end_date of versioned results is None.

### CODE REVIEW

**Review by dmnapolitano — COMMENTED** [review]
Thanks for these changes! 🎉  I also noticed there's no mention of how to use the extrapolation rule in the README.  Can you add something? 🤔

**Comment by lennybronner** [comment]
> Thanks for these changes! 🎉 I also noticed there's no mention of how to use the extrapolation rule in the README. Can you add something? 🤔

good call. readme updated!

**Review by dmnapolitano — APPROVED** [review]
Yay thank you!!! 🎉 🎉

File: src/elexmodel/handlers/data/VersionedData.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import logging
```
Comment (dmnapolitano) [inline src/elexmodel/handlers/data/VersionedData.py:1]: Should this be `from elexmodel.logger import getModelLogger`? 🤔

Comment (lennybronner) [reply src/elexmodel/handlers/data/VersionedData.py]: yeah it should, good catch, thanks. unclear how that happened

**Pull request merged**